nemo/collections/nlp/models/language_modeling/megatron_bert_model.py

```diff
@@ -306,6 +310,15 @@ def build_pretraining_data_loader(self, dataset, consumed_samples):
         )
 
     def setup(self, stage=None):
+        resume_checkpoint_path = self.trainer.checkpoint_connector.resume_from_checkpoint_fit_path
+        if resume_checkpoint_path:
+            init_consumed_samples = int(
```

Comment:
Should we wrap in `try except` and provide a meaningful error if reading consumed samples fails?

Reply:
Good idea.

Reply:
I added the try-except to spit out the message if the numbers cannot be parsed.